docs/operator/deploy.md

```diff
@@ -138,18 +141,30 @@ drand start --tls-disable
 
 ### Test the connection to a node
 
-Use `drand util check <address>` to test the gRPC endpoint a drand node.
+Use `drand util check <address>` to test the gRPC endpoint on a drand node.
+
+```bash
+drand util check example.com --id <beacon-id>
+
+> drand: id example.com answers correctly
+```
+
+If the address used is a DNS name, this command will try to resolve the DNS name to IP.
+
+### Test the connection to a network
+
+Use `drand util check <address> --id <beacon-id>` to test the gRPC endpoint of a drand network which has a specific beacon id.
 
 ```bash
-drand util check example.com
+drand util check example.com --id <beacon-id>
```

Comment:
There is no difference with the previous section, is there ?

Reply:
Yes, you are right. That --id flag should not be there!